src/ads/BannerAd.tsx

```diff
@@ -16,9 +16,15 @@
  */
 
 import React from 'react';
+import { NativeModules } from 'react-native';
 import { BannerAdProps } from '../types/BannerAdProps';
 import { BaseAd } from './BaseAd';
 
 export function BannerAd({ size, ...props }: BannerAdProps) {
   return <BaseAd sizes={[size]} {...props} />;
 }
+
+export function getCurrentOrientationAnchoredAdaptiveBannerHeight() {
+  const { RNAppModule } = NativeModules;
+  return RNAppModule.getCurrentOrientationAnchoredAdaptiveBannerHeight();
```

Comment:
If you call into the native module without a platform check, and without an iOS implementation, this will cause a redbox error / crash on iOS. Is there a default implementation we should implement for non-android platforms? Or do we need an implementation for iOS as well?

Reply:
Yes, We should implement this for IOS too, but since I am not familiar with the objective-s and did not know whether the feature would be in demand or rejected, I saved my energy

Reply:
I'll wait to see what others think to see if they also like it then, so as not to waste any time if I'm the only fan :-)

Reply:
We also already have a similar static method: https://github.com/invertase/react-native-google-mobile-ads/blob/main/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsCommon.java#L43